src/main/ts/Part2Ex4FP.ts

```diff
@@ -149,17 +159,22 @@ Again, this looks familiar from our getOrElse1 function above.
 
 TODO: rewrite getOrElse1 using both Fun.identity and the "constant" function defined above.
  */
-
+export const getOrElse3 = <A> (oa: Optional<A>, other: A): A =>
+  oa.fold(
+    Fun.constant(other),
+    (a) => Fun.identity(a)
```

Comment:
```suggestion
    Fun.identity
```
Same as Lee commented above, this is just another eta reduce thing. It's essentially just doing the same thing you did on the line above, by writing `Fun.constant(other)` directly instead of `() => Fun.constant(other)()`